src/App/App.tsx

```diff
@@ -31,65 +28,61 @@ export default function App() {
   const SentryRoutes = Sentry.withSentryReactRouterV6Routing(Routes);
 
   return (
-    <WalletProvider {...chainOptions}>
```

Comment:
move wallet contexts to donate page, will probably return to top level if to be used in another route e.g giftcards